src/executor/logical_tile.cpp

```diff
@@ -164,7 +164,7 @@ storage::Tile *LogicalTile::GetBaseTile(oid_t column_id) {
 type::Value LogicalTile::GetValue(oid_t tuple_id, oid_t column_id) {
   PL_ASSERT(column_id < schema_.size());
   PL_ASSERT(tuple_id < total_tuples_);
-  PL_ASSERT(visible_rows_[tuple_id]);
+  //PL_ASSERT(visible_rows_[tuple_id]);
```

Comment:
This was there as I was unsure to remove the assertion. But as requested, it was removed.